app/main/forms/suppliers.py

```diff
@@ -28,20 +29,43 @@ class EditSupplierForm(Form):
 class EditContactInformationForm(Form):
     id = IntegerField()
     contactName = StripWhitespaceStringField('Contact name', validators=[
-        DataRequired(message="You must provide a contact name"),
+        InputRequired(message="You must provide a contact name"),
     ])
     email = EmailField('Contact email address', validators=[
-        DataRequired(message="You must provide an email address"),
+        InputRequired(message="You must provide an email address"),
```

Comment:
Do we want to add Length validators to these (and other similar places) as well?

Reply:
We could. Not sure it's part of this ticket particularly though. We have other tickets that will touch these.

Reply:
Sure, although changing them to InputRequired isn't part of this ticket either. If we don't do it here, let's make sure we make a ticket to add these validators so we don't forget. It can probably be a tech debt ticket (one of us should just slap our name on it straight away probably though, so it doesn't get lost). Happy to do it if you just want to Ship The Thing.

Reply:
Yeah I guess. Changing to InputRequired was a two second find and replace, adding validators is a bit more involved I think due to the tests involved and making sure the content of the messages was right etc. Feels like more creep.

I ❤️🚢

Reply:
Yeah that's fair, it's a bit creepy.